addons/website_event_crm/models/event_registration.py

```diff
@@ -26,6 +26,32 @@ def _get_lead_description_fields(self):
         res.append('registration_answer_ids')
         return res
 
+    def _get_lead_grouping(self, rules, rule_to_new_regs):
```

Comment:
Note to me:
Not sure if we need to group by visitor here, as we could probably just take the lang of the first visitor in `_get_lead_values()`
(Seems also weird to have a group of registrations with different visitor, but perhaps weird thing happens when merging visitor ?)